### ISSUE TITLE
Import interval are wrong

### ISSUE THREAD

**Issue by clochix** [cozy-labs/konnectors#337]
Users are reporting on [the forum](https://forum.cozy.io/t/app-konnectors-ne-suit-pas-les-regles-dimportation-automatique/2815) and IRC that import interval doesn't seem to work.

**Comment by cnovel** [comment]
Here's the log when it tries to update each hour
```
[2016-06-28 11:22:35:534] info - birthdays: Next update 28/06/2016 at 12:22:35
[2016-06-28 11:23:50:23] info - ical_feed: Next update 28/06/2016 at 12:23:50
[2016-06-28 12:22:35:553] info - birthdays: Next update 28/06/2016 at 13:22:35
[2016-06-28 12:23:50:33] info - ical_feed: Next update 28/06/2016 at 13:23:50
[2016-06-28 13:22:35:573] info - birthdays: Next update 28/06/2016 at 14:22:35
[2016-06-28 13:23:50:45] info - ical_feed: Next update 28/06/2016 at 14:23:50
[2016-06-28 14:22:35:587] info - birthdays: Next update 28/06/2016 at 15:22:35
[2016-06-28 14:23:50:65] info - ical_feed: Next update 28/06/2016 at 15:23:50
[2016-06-28 15:22:35:601] info - birthdays: Next update 28/06/2016 at 16:22:35
[2016-06-28 15:23:50:81] info - ical_feed: Next update 28/06/2016 at 16:23:50
[2016-06-28 16:22:35:607] info - birthdays: Next update 28/06/2016 at 17:22:35
[2016-06-28 16:23:50:83] info - ical_feed: Next update 28/06/2016 at 17:23:50
[2016-06-28 17:22:35:622] info - birthdays: Next update 28/06/2016 at 18:22:35
[2016-06-28 17:23:50:94] info - ical_feed: Next update 28/06/2016 at 18:23:50
[2016-06-28 18:22:35:630] info - birthdays: Next update 28/06/2016 at 19:22:35
[2016-06-28 18:23:50:99] info - ical_feed: Next update 28/06/2016 at 19:23:50
[2016-06-28 19:22:35:631] info - birthdays: Next update 28/06/2016 at 20:22:35
[2016-06-28 19:23:50:98] info - ical_feed: Next update 28/06/2016 at 20:23:50
[2016-06-28 20:22:35:649] info - birthdays: Next update 28/06/2016 at 21:22:35
[2016-06-28 20:23:50:121] info - ical_feed: Next update 28/06/2016 at 21:23:50
```
It seems it just postpones the update but doesn't actually do anything.

**Comment by frankrousseau** [comment]
It may be related to the fact that konnectors handle several accounts now. The change is probably not properly handled on the scheduler side. I will have a look at it.

**Comment by ZeHiro** [comment]
looks like this is due do the multiple accounts thing : 
https://github.com/cozy-labs/konnectors/blob/master/server/init/konnectors.coffee#L18

**Comment by ZeHiro** [comment]
@cnovel It seems to work on konnectors 0.7.0 . And I observe it is working again on my cozy. 
Fixed by : ebce19ece863790f38c90dbcd170f239946e0c52
